packages/suite/src/views/recovery/index.tsx

```diff
@@ -138,20 +138,28 @@ export const Recovery = ({ onCancel }: ForegroundAppProps) => {
     );
 
     const getStep = () => {
+        const isShamirBackupAvailable =
+            device?.features?.capabilities?.includes('Capability_Shamir');
+
+        // Shamir backup uses 20 and 33 word shares
+        const seedBackupLengthMessage = isShamirBackupAvailable
+            ? 'TR_SEED_BACKUP_LENGTH_INCLUDING_SHAMIR'
+            : 'TR_SEED_BACKUP_LENGTH';
+
         switch (recovery.status) {
             case 'initial':
                 return (
                     <>
                         <LeftAlignedP>
-                            {model && <Translation id={`TR_CHECK_RECOVERY_SEED_DESC_T${model}`} />}
+                            <Translation id={seedBackupLengthMessage} />
```

Comment:
There was a different message here: `Your recovery seed (wallet backup) is entered using the Trezor Model T touchscreen. This avoids exposing any of your sensitive information to a potentially insecure computer or web browser.`

Reply:
18434f8917479958e2bd2632b4771ce9c0e521cd
53d4ffeaa16c9b27fee8675829886b13fa12766a